README.md

```diff
@@ -16,66 +16,69 @@ This repository is for Browser.html (the front-end). Active development of Graph
 We welcome contributions from anyone. See [CONTRIBUTING.md](https://github.com/browserhtml/browserhtml/blob/master/CONTRIBUTING.md) for help getting started.
 
 
-### Building and Running
+## Development
 
-The Browser.html UI is bundled with Servo. To run it, you'll need to build Servo.
+There are two major components to the browser.html application.
 
-First, [install Servo's prerequisites](https://github.com/servo/servo#setting-up-your-environment). Then, clone and build Servo:
+1. Local server that serves application UI.
+2. Client that is a application shell that connects to the server and renders served UI.
 
-``` sh
-git clone https://github.com/servo/servo
-cd servo
-./mach build -r
-```
-
-Finally, run Servo with the `--browserhtml` flag.
 
-``` sh
-./mach run -r --browserhtml
-```
+### Server
 
-
-### Developing the Front-end
-
-If you're working on the Browser.html front-end, you'll want to run the web app locally.
-
-**Prerequisites**: You'll need [Node](https://nodejs.org/) and NPM. Next, clone Browser.html and install its Node dependencies.
+If you're working on the Browser.html front-end, you'll need a [node][] to install all dependcies and run development toolchain.
 
 ``` sh
 git clone https://github.com/browserhtml/browserhtml.git
 cd browserhtml
 npm install --no-optional
 ```
 
-Then, start the front-end local server:
+Once all dependencies are installed, you can run a server component with following command:
 
 ``` sh
 npm run build-server
 ```
 
-Finally, in a new command line interface application (ex: Terminal), start Servo with the Browser.html flags turned on in either debug (`-d`) or release (`-r`) mode:
+Any changes to the source code will trigger a build, once complete server will serve a new version. That will allow you to reload a client in order to see your changes. Alternatively you could run a "live server" which also re-builds on changes, but in addition it also provides live code reload for the UI such that application state is preserves and no reload of the client is required. You can run "live server" with a following command:
```

Comment:
How about this slightly changed version:

Any changes to the source code will trigger a build, which is then automatically served. That will allow you to reload a client in order to see your changes. Alternatively you can use `live-server` to not only rebuild, but also trigger an automatic live code reload for the UI such that application state is preserved: